Content.Server/FlavorText/FlavorTextComponent.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Content.Server.FlavorText
+{
+    [RegisterComponent]
+    public sealed class FlavorTextComponent : Component
```

Comment:
this is mostly a semantics thing but i think this should be something like `DetailedDescriptionComponent` so we can use it for other stuff that isnt necessarily flavor text